packages/webview_flutter/example/lib/main.dart

```diff
@@ -53,26 +77,31 @@ class WebViewExample extends StatelessWidget {
           return Container();
         });
   }
+
+  Future<dynamic> _runSampleJSEvaluation(WebViewController controller) async {
+    // "document.body.setAttribute('style','background: red')
+    return controller.evaluateJavaScript(
+        "document.body.setAttribute('style','background: red');document.body.style.backgroundColor;");
+  }
 }
 
 class SampleMenu extends StatelessWidget {
-  const SampleMenu();
+  SampleMenu({this.onSelected});
```

Comment:
I think injecting the controller future into here and have this widget return a FutureBuilder in it's build method would be simpler.